### ISSUE TITLE
About the attention model

### ISSUE THREAD

**Issue by wcj-Ford** [shmsw25/mcb-model-for-vqa#2]
Hi, I`m reading your code and have one question.
 # end for First MCB module
        signed_att = tf.sign(att)*tf.sqrt(tf.abs(att))
        normalized_att = tf.nn.l2_normalize(signed_att, 0)
        normalized_att = tf.transpose(normalized_att, [0, 2, 3, 1])
Why there is a signed_att and tf.nn.l2_normalize layer?
Thank you!
